subsys/mgmt/mcumgr/grp/os_mgmt/src/os_mgmt.c

```diff
@@ -471,7 +471,7 @@ os_mgmt_bootloader_info(struct smp_streamer *ctxt)
 		     zcbor_int32_put(zse, BOOTLOADER_MODE);
 #if IS_ENABLED(MCUBOOT_BOOTLOADER_NO_DOWNGRADE)
 		ok = zcbor_tstr_put_lit(zse, "no-downgrade") &&
-		     zcbor_bool_encode(zse, true);
+		     zcbor_bool_encode(zse, &true);
```

Comment:
strange - true is not a variable or so

Reply:
Compiler loads true on stack and gets pointer to stack, it works, although I can make this more clear.

Reply:
this throws a compiler error